backend/xivdpscalc/pps/sch.py

```diff
@@ -2,7 +2,27 @@
 
 import math
 
+from collections import defaultdict
+from xivdpscalc.character import Character
 from xivdpscalc.pps import HealerPps
+from xivdpscalc.pps.sch_action import SchAction, SchEffect, SchResource, SchSimNotice
+from xivdpscalc.types import ElapsedTime
+from xivdpscalc.pps.rotation import SchRotation
+
+SchSimTimeline = dict[SchAction, list]
+
+class SchSimResults:
```

Comment:
Would this be better served as a `dataclass` since the arguments are just initialized with no further logic?